inputs/demand/industry/industry_useful_demand_for_chemical_other_electricity_efficiency.ad

```diff
@@ -1,10 +1,10 @@
 - query = UPDATE(V(industry_useful_demand_for_chemical_other_electricity), preset_demand, NEG(USER_INPUT()))
 - priority = 0
-- max_value = 10.0
-- min_value = -10.0
+- max_value = 100.0
+- min_value = 0.0
```

Comment:
Note that previously, the efficiency gain was also able to become negative. By setting the min_value at 0.0 you remove this possibility from scenarios. This can break existing scenarios where a negative efficiency gain is set. Also note that a min_value of -10%/y change can lead to quite high values, for 2019 to 2040 it for example leads to a change of 740%: 1.1 ^ (2040-2019).

Avoiding breaking of existing scenarios requires a data migration and potentially a new min value.

Reply:
After a discussion and quick review on the amount of scenario's, @mabijkerk and I decided to leave the min_value at 0.0 for consistency sake (all other efficiency sliders also start at 0.0).